dlk/examples/classification/lmnet_quantize_cifar10/config.yaml

```diff
@@ -24,7 +24,6 @@ DATASET:
       is_resize: false
       width: 32
     - !!python/object:lmnet.data_augmentor.FlipLeftRight
-      is_bounding_box: false
```

Comment:
I just wonder is it okay to remove `is_bounding_box` line without removing `probability` line?
Why not removing both of them?

Reply:
@lm-jira
I removed only `is_bounding_box` parameter, don’t remove `probability` parameter. `probability` still remain.